### DISCUSSION TITLE
Add import camera motion option

### DISCUSSION THREAD

**Discussion by vitos1k** [bezo97/IFSRenderer#34] [Ideas]
import fbx,alembic or CSV list of camera keyframes for animations of camera. That would help to integrate your tool into  pipelines to tie up with existing camera animations from other DCCs

**Comment by bezo97** [comment]
Great idea! The CSV import could be implemented with relatively small effort. Would you be able to provide a sample CSV file?

**Reply by vitos1k** [reply]
sure. file is in the attachment. CSV format: object_name, frame, position x, position y, position z, scale x, scale y,scale z, roll, pitch, yaw

Roll Pitch Yaw are in radians
Though there is no info on framerate in the csv, would it be crucial? it's was set to 24fps
I used this addon for export:
https://gist.github.com/astronotter/faec735fc5e270df57741705052096ac

i can modify addon and add aditional info into csv if you need.
[example.csv](https://github.com/bezo97/IFSRenderer/files/15417448/example.csv)

**Reply by bezo97** [reply]
Thanks I can work with this. The framerate is not crucial, the user will just have to set the framerate to the same value in IFSRenderer as in the external program.

**Reply by vitos1k** [reply]
I noticed that camera Navigation panel has 4 fields for Orientation i assume those are quaternions. I have re-exported CSV file with quaternions for you to use
[example2.csv](https://github.com/bezo97/IFSRenderer/files/15425107/example2.csv)
